### DISCUSSION TITLE
Not syncing

### DISCUSSION THREAD

**Discussion by gpsrai09** [Chia-Network/chia-blockchain#4349] [Q&A]
My mining stopped syncing by itself. I restart it but it still same .
How can I fix it 
Please help

Thanks

**Comment by angelux813** [comment]
I have the same problem.  Anyone knows how to fix it?    OS Win 10

**Comment by lvcivs** [comment]
This usually means that you don't have enough connections. Try the recommendations on the following FAQ entry: https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#why-does-my-node-have-no-connections-how-can-i-get-more-connections
